homeassistant/components/kef/media_player.py

```diff
@@ -117,6 +118,7 @@ async def async_setup_platform(
     )
 
     mode = get_ip_mode(host)
+    getmac.getmac.FORCE_METHOD="ArpFile"
```

Comment:
This should be fixed in the `getmac` library as already discussed here:

https://github.com/GhostofGoes/getmac/issues/83

Regarding the different versions of the library, we can denylist `get-mac` in `package_constraints` like this:

https://github.com/home-assistant/core/blob/1431979eab9cfe158deaba9ccf268f8315abe7a6/homeassistant/package_constraints.txt#L82

Libraries that use `get-mac` should be updated to use `getmac` instead so we eventually can remove the denylist item.

Reply:
You are right. I have asked the dev of one of the ha integrations to update that package in ha to use getmac. I'll add the package constraint in this patch.